js/statuspageApi.js

```diff
@@ -9,7 +11,40 @@ class StatuspageApi {
   }
 
   lastIncident() {
-    return this.data.incidents[0];
+    let chosenIncident = undefined;
+    let incidents = this.data.incidents[0];
+    let scheduledMaintenances;
+    let incidentUpdated;
+    let maintenanceUpdated;
+
+    // if scheduled_maintenances has a relevant hashtag, add to the scheduledMaintenances variable
+    // if not, the chosenIncident is automatically incidents
+    if (
+      this.data.scheduled_maintenances === undefined ||
+      !hashtagRegexp.test(
+        this.data.scheduled_maintenances[0].incident_updates[0].body
+      )
```

Comment:
So this logic in the "if" block could be its own method, something descriptive like `doesScheduledMaintenanceMatchHashtag` and/or it'd be nice to also refactor `hasMatchingHashtag` so that they share the logic that checks against the regexp.